### DISCUSSION TITLE
What does MetaTable do in rootcoord?

### DISCUSSION THREAD

**Discussion by ArenaSu** [milvus-io/milvus#33401] [Q&A and General discussion] [answered]
I want to add comments for  MetaTable in rootcoord, but i can not explain it well.The description below maybe ok?

MetaTable is a metadata set for databases, collections, partitions and relationships between them.

**Comment by xiaofan-luan** [comment, marked as answer]
agree on that part. 

meta table persistent all the meta info of databases, collections and partitions.

**Reply by ArenaSu** [reply]
Is MetaTable stored persistently ? Or MetaTable is just extracted from multiple pieces of kv data such as catalog?

**Reply by xiaofan-luan** [reply]
Meta table persist data in kv and reconcile it in memory each time in failure recovery.
